src/DependencyInjection/Configuration.php

```diff
@@ -280,6 +280,15 @@ private function createClientsNode() : ArrayNodeDefinition
                                     ->thenInvalid('version can be: string or float')
                                 ->end()
                             ->end()
+                            ->scalarNode('handler')
```

Comment:
I think it's better to put `handler` option on the same level with `base_url` and `class`. It will be strange to redefine class of client on one level but class of handler on another.

Reply:
Done.